src/Rules/Functions/ExistingClassesInTypehintsRule.php

```diff
@@ -38,11 +38,11 @@ public function processNode(Node $node, Scope $scope): array
 			$node->getOriginalNode(),
 			$scope->getFunction(),
 			sprintf(
-				'Parameter $%%s of function %s() has invalid typehint type %%s.',
+				'Parameter $%%s of function %s() has invalid type %%s.',
 				$functionName
 			),
 			sprintf(
-				'Return typehint of function %s() has invalid type %%s.',
+				'Return type of function %s() has invalid type %%s.',
```

Comment:
maybe

```suggestion
				'function %s() has invalid return type %%s.',
```

Reply:
That would match the suggestion I have for anonymous functions.